packages/strapi-plugin-content-manager/config/policies/has-draft-and-publish.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = (ctx, next) => {
+  const {
+    params: { model: modelUid },
+  } = ctx;
+
+  const model = strapi.contentTypes[modelUid];
+  const hasDraftAndPublish = _.get(model, 'options.draftAndPublish', false);
```

Comment:
Once the PR `add published_at` is merge, the util function `hasDraftAndPublish` will be available to use, I think it would be better to use it then :)